lib/Models/Catalog/Ows/WebMapServiceCapabilities.ts

```diff
@@ -132,7 +132,7 @@ type Mutable<T> = { -readonly [P in keyof T]: T[P] };
 export function getRectangleFromLayer(
   layer: CapabilitiesLayer
 ): StratumFromTraits<RectangleTraits> | undefined {
-  var egbb = layer.EX_GeographicBoundingBox; // required in WMS 1.3.0
+  var egbb = layer?.EX_GeographicBoundingBox; // required in WMS 1.3.0
```

Comment:
If the `layer` can be `undefined`, it is not reflected in the type here. Do we really need `layer?.`?

Reply:
Good point! I have removed the optional chaining